charts/rpe-send-letter-service/values.preview.template.yaml

```diff
@@ -2,7 +2,6 @@ java:
   # Don't modify below here
   image: ${IMAGE_NAME}
   ingressHost: ${SERVICE_FQDN}
-  aadIdentityName:
```

Comment:
had to include changes here as either pipeline or dependency check will fail otherwise

Reply:
not sure why 2 individual PRs attempting to solve this kept on failing on AKS build :tired_face: :man_shrugging: